app/components-react/shared/Tabs.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { CSSProperties } from 'react';
+import { Tabs as AntdTabs } from 'antd';
+import { $t } from 'services/i18n';
+
+interface TabData {
+  label: string | JSX.Element;
+  key: string;
+}
+
+interface ITabs {
+  data?: TabData[];
+  onChange?: (param?: any) => void;
+  style?: CSSProperties;
+  tabStyle?: CSSProperties;
+}
+
+export default function Tabs(p: ITabs) {
+  const dualOutputData = [
+    {
+      label: (
+        <span>
+          <i className="icon-desktop" style={{ paddingRight: '5px' }} />
+          {$t('Horizontal')}
+        </span>
+      ),
+      key: 'horizontal',
+    },
+    {
+      label: (
+        <span>
+          <i className="icon-phone-case" style={{ paddingRight: '5px' }} />
+          {$t('Vertical')}
+        </span>
+      ),
+      key: 'vertical',
+    },
+  ];
```

Comment:
this seems weird to put into a shared component, not sure how much we're gaining by extracting this